ddl/sequence_test.go

```diff
@@ -1088,3 +1090,17 @@ func (s *testSequenceSuite) TestAlterSequencePrivilege(c *C) {
 	c.Assert(err.Error(), Equals, "[planner:1142]ALTER command denied to user 'myuser'@'localhost' for table 'my_seq'")
 	tk.MustExec("drop sequence if exists my_seq")
 }
+
+func TestDdl_AlterSequenceIssue31265(t *testing.T) {
```

Comment:
This case is different from the issue. Try to set it no-cache and then set the cache size of it. The result should be cached.